.gitignore

```diff
@@ -18,3 +18,7 @@ src/.vs/*
 
 # PostBuildEvents.bat will differ for all contributors
 src/Our.Umbraco.RobotsTxtEditor/PostBuildEvents.bat
+
+# generated by build process
+artifacts/**
+build/tools/vswhere*
```

Comment:
What is "vswhere"? 🎵 _"...tell me more, tell me more..."_ 🎵
Can we include it in the repo? (e.g. is it a small or large program?)

Reply:
vswhere is a 'Visual Studio Locator' tool (https://github.com/microsoft/vswhere).  It is downloaded by nuget during the build process so it can find the location of the MSBuild tool in both VS2017 and VS2019 (regardless of edition, e.g. community, pro etc)

The changes done in this commit were done with guidance from @jeavon!

Reply:
@emmaburstow and I updated the properties of the build project. 

I then opened the solution folder in the command prompt and ran `build.cmd`.  When prompted for version numbers etc I just entered some random numbers as didn't know what they should be. The end result was some files created in the 'artifacts' folder.  

### Umbraco Package

I tested the Umbraco project by installing the created .zip file as a local package in a new v8 website:

![local-package-1](https://user-images.githubusercontent.com/4716542/61190308-323ff880-a692-11e9-8ce8-89033e6d1159.PNG)

![local-package-2](https://user-images.githubusercontent.com/4716542/61190309-323ff880-a692-11e9-9289-894081575909.PNG)

I then uninstalled the package via the back office.  All worked as expected.

### Nuget Package

I copied the nuget (.nupkg) files to a location on my machine and added this folder to the nuget locations in my visual studio (Tools > Options > Nuget Package Manager > Package Sources). Then should be able to install the package via nuget:

![nuget-package-1](https://user-images.githubusercontent.com/4716542/61190310-32d88f00-a692-11e9-863c-737fdcd5d885.PNG)

![nuget-package-2](https://user-images.githubusercontent.com/4716542/61190306-323ff880-a692-11e9-8c81-4a721c59c1cc.PNG)

![nuget-package-3](https://user-images.githubusercontent.com/4716542/61190307-323ff880-a692-11e9-811d-39ff2da6de89.PNG)

So I think this all works! Just depends whether @leekelleher you agree on what the settings (in terms of author, license etc) should be.

Reply:
> vswhere is a 'Visual Studio Locator' tool (https://github.com/microsoft/vswhere). It is downloaded by nuget during the build process so it can find the location of the MSBuild tool in both VS2017 and VS2019 (regardless of edition, e.g. community, pro etc)

Thanks for the info @LottePitcher - everyday is a school day! 😃 
Soooo... how does the NuGet package get put in the "`build/tools/vswhere`" folder?